### PR TITLE
Fix compiler warnings (clang 8)

### PR DESCRIPTION
**Pull request by havogt**
## Technical Description

Fix warnings for clang 8.

####  Enhances

No compiler warnings when compiling with clang 8.

## Dependencies

This PR is independent

### CODE REVIEW

**Comment by havogt** [comment]
launch pipeline

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@
 *.user
 *.project
 .idea
+.vscode
```
Comment (twicki) [inline .gitignore]: would it make sense to have a section about vscode with multiple folders / files in there?

File: src/dawn/IIR/IIR.h [hunk 2]
```diff
@@ -87,8 +87,8 @@ class IIR : public IIRNode<void, IIR, Stencil> {
 
   const sir::GlobalVariableMap& getGlobalVariableMap() const { return globalVariableMap_; }
 
-  void insertGlobalVariable(std::string name, std::shared_ptr<sir::Value> value) {
-    globalVariableMap_.emplace(name, value);
+  void insertGlobalVariable(std::string var_name, std::shared_ptr<sir::Value> value) {
```
Comment (twicki) [inline src/dawn/IIR/IIR.h]:
```suggestion
  void insertGlobalVariable(std::string varName, std::shared_ptr<sir::Value> value) {
```

File: src/dawn/IIR/IIR.h [hunk 3]
```diff
@@ -87,8 +87,8 @@ class IIR : public IIRNode<void, IIR, Stencil> {
...
+    globalVariableMap_.emplace(var_name, value);
```
Comment (twicki) [inline src/dawn/IIR/IIR.h]:
```suggestion
    globalVariableMap_.emplace(varName, value);
```

File: .gitignore (hunk 1 above)
Comment (havogt) [reply .gitignore]: Let me remove it from there and we take it from your PR

Comment (havogt) [inline .gitignore]:
```suggestion
```

**Comment by havogt** [comment]
launch pipeline

**Comment by havogt** [comment]
jenkins is more or less green...

**Pull request merged**